src/components/about/About.css

```diff
@@ -0,0 +1,58 @@
+p{ 
+    margin:0;
```

Comment:
Don't forget about space after :. It's related for all rows in this file (11, 42, 43, etc)
e.g `margin: 0`.